go.mod

```diff
@@ -31,45 +31,46 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 )
 
+require k8s.io/kubernetes v1.29.1
```

Comment:
could you please revert the updates of the dependencies?

Reply:
yes done